### PR TITLE
Fixed redirect by question code

### PR DESCRIPTION
**Pull request by weslyvo81**

### CODE REVIEW

**Review by jasperRob — APPROVED** [review]
LGTM

File: kogitoq/kogito-common/src/main/java/life/genny/kogito/common/service/NavigationService.java [hunk 1]
```diff
@@ -12,9 +12,11 @@
 import javax.inject.Inject;
 import javax.json.Json;
 import javax.json.JsonObject;
+import javax.json.JsonObjectBuilder;
 import javax.json.bind.Jsonb;
 import javax.json.bind.JsonbBuilder;
 
+import life.genny.qwandaq.utils.*;
```
Comment (varunshastry) [inline kogitoq/kogito-common/src/main/java/life/genny/kogito/common/service/NavigationService.java]: Wild-card import to be removed.

**Review by adamcrow64 — APPROVED** [review]
LGTM. I need to use CommonUtils more.

**Review by varunshastry — APPROVED** [review]

File: kogitoq/kogito-common/src/main/java/life/genny/kogito/common/service/NavigationService.java (hunk 1 above)
Comment (weslyvo81) [reply kogitoq/kogito-common/src/main/java/life/genny/kogito/common/service/NavigationService.java]: resolved

**Pull request merged**